README.md

```diff
@@ -2,50 +2,91 @@
+<br></br>
 
-You can configure whether or not tiling resources are included in deployment by setting the `deploy_tiling` flag in the [inputs](./inputs.tf) file.
+### IO
+
+The IO module includes [MinIO](https://min.io/), an S3 compatible object storage for the usage of the [SWOOP API](https://github.com/Element84/swoop), [SWOOP Caboose](https://github.com/Element84/swoop-go), and [SWOOP Conductor](https://github.com/Element84/swoop-go). More information is available [here](./modules/io/README.md).
+
+You can configure whether or not STAC-FastAPI is included in deployment by setting the `deploy_minio` flag in the [inputs](./inputs.tf) file.
+<br></br>
+
+### DB
+
+The DB module includes [PostgreSQL](https://www.postgresql.org/), an open source object-relational database system for the usage of the [SWOOP API](https://github.com/Element84/swoop), [SWOOP Caboose](https://github.com/Element84/swoop-go), and [SWOOP Conductor](https://github.com/Element84/swoop-go). More information is available [here](./modules/db/README.md).
+
+You can configure whether or not Postgres is included in deployment by setting the `deploy_postgres` flag in the [inputs](./inputs.tf) file.
+
+The DB module also has a configurable swoop-db-init which will initialize [swoop-db](https://github.com/Element84/swoop-db) with the creation of a SWOOP database and Roles for each SWOOP Component. You can configure whether or not swoop-db-init is included in deployment by setting the `deploy_db_init` flag in the [inputs](./inputs.tf) file.
+<br></br>
 
 ### STAC-FastAPI
 
 The local development environment includes a module for deploying STAC-FastAPI, which provides an API that exposes a STAC catalog that is persisted in a backend Postgres database with its component STAC Collections and Items. More information is available [here](./modules/stac-fastapi/README.md).
 
 You can configure whether or not STAC-FastAPI is included in deployment by setting the `deploy_stacfastapi` flag in the [inputs](./inputs.tf) file.
+<br></br>
+
+### SWOOP
+
+The swoop module contains the SWOOP Bundle which will install the followin the following components into a kubernetes cluster:
+* [swoop-api](https://github.com/Element84/swoop) - configure whether or not SWOOP-API is included in deployment by setting the `deploy_swoop_api` flag in the [inputs](./inputs.tf) file
+* [argo-workflows](https://github.com/argoproj/argo-workflows/) - configure whether or not SWOOP-API is included in deployment by setting the `deploy_argo_workflows` flag in the [inputs](./inputs.tf) file
+* [swoop-caboose](https://github.com/Element84/swoop-go) - configure whether or not SWOOP-API is included in deployment by setting the `deploy_swoop_caboose` flag in the [inputs](./inputs.tf) file
+* [swoop-conductor](https://github.com/Element84/swoop-go) - configure whether or not SWOOP-API is included in deployment by setting the `deploy_swoop_conductor` flag in the [inputs](./inputs.tf) file
```

Comment:
"whether or not Swoop Conductor is included"

Reply:
fixed, thanks for catching that